roles/upgrade/tasks/leapp-post-upgrade-selinux.yml

```diff
@@ -22,9 +24,11 @@
   when: selinux_results.reboot_required
 
 - name: leapp-post-upgrade-selinux | Verify SELinux is set to {{ selinux_mode }}
-  ansible.posix.selinux:
-    policy: targeted
-    state: "{{ selinux_mode }}"
+  ansible.builtin.include_role:
+    name: fedora.linux_system_roles.selinux
+  vars:
+    selinux_policy: targeted
+    selinux_state: "{{ selinux_mode }}"
   check_mode: true
   register: selinux_check_results
   failed_when: selinux_check_results.changed
```

Comment:
include_role might not return `changed` variable

Reply:
in fact - you cannot use `check_mode`, `register`, or `failed_when` with `include_role`